front/src/applications/stdcm/hooks/useStaticPathfinding.ts

```diff
@@ -72,6 +77,7 @@ const useStaticPathfinding = (infra?: InfraWithState) => {
         infraId: infra.id,
         rollingStock,
         pathSteps: pathStepsLocations,
+        loadingGauge: loadingGauge || undefined,
```

Comment:
This is enough as `loadingGauge` can already be undefined
```suggestion
        loadingGauge,
```

Reply:
Done